gamestonk_terminal/cryptocurrency/discovery/discovery_controller.py

```diff
@@ -0,0 +1,232 @@
+"""Cryptocurrency Overview Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302, W0622
+import argparse
+import os
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency.discovery import (
+    pycoingecko_view,
+    coinpaprika_view,
+    coinmarketcap_view,
+    binance_view,
+)
+
+
+class Controller:
+
+    CHOICES = [
+        "?",
+        "cls",
+        "help",
+        "q",
+        "quit",
+        "cp_search",
```

Comment:
So, one of the things we are going to write on the Coding Guidelines is commands to not have underscores, since that requires 2 fingers to type, and adds latency to execution speed.

Could we have these so that they have no underscore _ ?

Reply:
Yes, for sure. I agree that it's not to user friendly to type _, but my question should I go with cpcoins, cgtrending and so on ? Or try to change all commands names to not have any prefixes like in previous version (there will be some overlap like with `coins` methods or `info` , `exchanges` and few others. Do you have anything on mind ?

Reply:
I think for now we could go with the cgcoins, cgtrending, ...

In the future, we can thinking on combining them. I think this is a futuristic step, for now having cg,cp at start without underscore is the right thing to do!